contentdm_exporter/contentdm_record_exporter.py

```
@@ -302,10 +338,12 @@ def get_file_level_id(elem):
     return file_level_id
 
 
-def get_file_metadata_xml(file_level_id, collection_alias):
+def get_file_metadata_xml(file_level_id, collection_alias, field_mapping={}):
```

Ruff (0.14.13)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)